R/priors.R

```diff
@@ -562,7 +561,22 @@ R2 <- function(location = NULL, what = c("mode", "mean", "median", "log")) {
   list(dist = "R2", location = location, what = what, df = 0, scale = 0)
 }
 
+#' @rdname priors
+#' @export
+default_prior_intercept = function() {
```

Comment:
I think `default_prior_*` should take a `family` argument and then `stan_glm`, etc. should default to `default_prior(family)`. This will give us more flexibility to change things in the future.

Reply:
Ok. What about for functions like `stan_lmer()`, `stan_glm.nb()`, `stan_glmer.nb()`, where there's no `family` argument? 

That is, for `stan_glmer()` we can have

```r
stan_glmer(..., prior_intercept = default_prior_intercept(family))
```

but for `stan_lmer()` and others should we just fill in the family like this?

```r
stan_lmer(..., prior_intercept = default_prior_intercept(family=gaussian()))
stan_glmer.nb(..., prior_intercept = default_prior_intercept(family=neg_binomial_2()))
stan_glm.nb(..., prior_intercept = default_prior_intercept(family=neg_binomial_2()))
```

Or some other option?